docs/storage/splunk.md

```diff
@@ -0,0 +1,55 @@
+# Exporting cAdvisor Stats to Splunk Enterprise or Splunk Cloud instance
+
+cAdvisor supports exporting stats to [Splunk Enterprise or Splunk Cloud instance](https://splunk.com). To use Splunk Enterprise or Splunk Cloud instance, you need to pass some additional flags to cAdvisor telling it where the Splunk instance is located:
+
+Set the storage driver as Splunk.
+
+```
+ -storage_driver=splunk
+```
+
+Specify what Splunk instance to push data to:
+
+```
+ # The *protocol://ip:port* of the Splunk Http Event Collector endpoint. Default is 'https://localhost:8088'
+ -storage_driver_splunk_url=https://ip:port
```

Comment:
Do you need "storage_driver".... before each param? why not just "splunk_url"? Is that the convention?